### PR TITLE
Add Ros2Pen static plugin

### LINKED ISSUE #766: Add Ros2Pen static plugin.
**Is your feature request related to a problem? Please describe.**
Add `Ros2Pen` static plugin. It should
* have ability to toggle `enable/disable` via `wb_pen_write`
* have ability to set color via `wb_pen_set_ink_color`

Use `Ros2LED` as a template.

**Describe the solution you'd like**

**Describe alternatives you've considered**
Considered implementing a custom driver like detailed [here](https://docs.ros.org/en/rolling/Tutorials/Advanced/Simulators/Webots/Setting-Up-Simulation-Webots-Basic.html#edit-the-my-robot-driver-plugin) or extending `webots_ros2_control` but those options seemed complicated.

**Additional context**

### PR DESCRIPTION
**Pull request by justinberi**
Closes #766

### CODE REVIEW

**Comment by justinberi** [comment]
Thanks. Will do.

File: webots_ros2_msgs/msg/PenInkProperties.msg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Set the ink properties of a pen device.
+# See https://www.cyberbotics.com/doc/reference/pen#wb_pen_set_ink_color for more details
+
+# Ink color in hexadecimal format
+int32 color
+
+# Ink density (similar in context to alpha of rgba)
+float32 density
```
Comment (ygoumaz) [inline webots_ros2_msgs/msg/PenInkProperties.msg:8]:
```suggestion
float32 density

```

Comment (ygoumaz) [reply webots_ros2_msgs/msg/PenInkProperties.msg:8]: Files should end with an empty line.

**Review by ygoumaz — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution. Just a very minor last detail and we can merge it.

**Review by ygoumaz — APPROVED** [review]

**Comment by ygoumaz** [comment]
Tests fail because no nightly build of Webots is available. They should pass tomorrow.

**Comment by justinberi** [comment]
Thanks for following up on this!

**Pull request merged**